src/Workspaces/Core/Portable/Extensions/AdditionalProperty.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq.Expressions;
+using System.Text;
+
+namespace Microsoft.CodeAnalysis
+{
+    internal class AdditionalProperty
+    {
+        public (string label, string value) propertyInfo;
```

Comment:
Why not have 2 string properties, `Name` and `Value` instead of exposing a tuple?